core/indexer/processTransactions.go

```diff
@@ -275,10 +275,10 @@ func (tdp *txDatabaseProcessor) groupNormalTxsAndRewards(
 		case block.RewardsBlock:
 			rTxs := getRewardsTransaction(txPool, mb.TxHashes)
 			for hash, rtx := range rTxs {
+				dbTx := tdp.commonProcessor.buildRewardTransaction(rtx, []byte(hash), mbHash, mb, header, mbTxStatus)
+				addToAlteredAddresses(dbTx, alteredAddresses, mb, selfShardID, true)
+				alteredAddresses[dbTx.Receiver] = struct{}{}
```

Comment:
This set is ok? It is done in the method addToAlteredAddresses called above only under some certain conditions. For case block.TxBlock: above this set is not done, this is why I thought that could be something wrong here or something missed above.

Reply:
removed, thanks